apm/monitor.go

```diff
@@ -25,6 +26,10 @@ type basicMonitor struct {
 // not automatically rotate data. The MonitorConfig makes it possible to
 // filter events. If this value is nil, no events will be filtered.
 func NewBasicMonitor(config *MonitorConfig) Monitor {
+	if config != nil && len(config.Tags) > 0 {
+		sort.Strings(config.Tags)
```

Comment:
What's the reason behind sorting these and in `(*eventWindows).Document()`? It's not an issue, I'm just curious.

Reply:
if we're going to ftdc compress this data, the order of the keys should be stable/consistent.